<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBInsertDlg</class>
 <widget class="QDialog" name="DBInsertDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增数据库数据</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,3,2,0">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>折弯机</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>机器人</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>对中台</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>翻面架</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>上料台</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>下料台</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>折弯机</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Machine">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editMachineName">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>外形长度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMachineLength_2">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>外形宽度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMachineLength_3">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>下夹具高度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMachineLength_4">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>外形宽度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMachineLength_5">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>开口高度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMachineLength_6">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>文件路径：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editMachinePath">
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnMachine">
            <property name="text">
             <string>文件选择</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Robot"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="whatsThis">
      <string>显示框</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(222, 222, 222);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAccept">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DBInsertDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
